src/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { sql, eq } from "drizzle-orm";
+import db from "./db";
+import { Config } from "./schemas/config";
+
+/**
+ * Casts a value into an array.
+ *
+ * @example
+ * castArray(1) // [1]
+ * castArray(['Hello', 'world']) // ['Hello', 'world']
+ */
+// rome-ignore lint/suspicious/noExplicitAny: can be an array of any type
+export const castArray = <T>(value: T): T extends any[] ? T : T[] => {
+	// rome-ignore lint/suspicious/noExplicitAny: can be an array of any type
+	return (Array.isArray(value) ? value : [value]) as T extends any[] ? T : T[];
+};
+
+/**
+ * Returns the config for a certain guild.
+ *
+ * @example
+ * getConfig('1')
+ */
+export const getConfig = (guildId: string) =>
+	db
+		.select()
+		.from(Config)
+		.where(eq(Config.id, sql.placeholder("guildId")))
+		.prepare()
+		.get({ guildId });
```

Comment:
You need to prepare the statement at startup and then run the prepared statement. Preparing it every time the query is executed removes the point of a prepared statement.